script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -34,15 +34,15 @@ If there were people in that building who hadn't yet evacuated, every single one
 <Page10>
 My body trembles at the unbelievable scene in front of me.
 I'm terrified at the prospect of what might happen next.
-It's just a coincidence that I'm still alive.//ruby dots <た|・><ま|・><た|・><ま|・>
+It's only by chance that I'm still alive.//ruby dots <た|・><ま|・><た|・><ま|・>
 If I had been behind Arcueid, I would've been frozen where I stood.
 <Page11>
 The expanding river of flames was a slow-moving disaster.
-People were running away faster than the fire was able to spread.
+People could run away faster than the fire was able to spread.
 But that glacier formed in an instant.
-Its range is small and the cold wave only crawls on the ground, but if one were to be caught by it, they would be killed without any chance to escape.
+Its range is narrow, and the cold wave only crawls on the ground, but once it's released there would be no chance of escaping it.
 <Page12>
-If that guy manages to get above ground, anyone in front of him would die without a chance to escape...!
+If that guy manages to get above ground, anyone that comes face to face with him would have no choice but to lay down and die.
```

Comment:
> もしヤツが地上に出てしまったら、ヤツの目の前にあるものは為す術なく息絶える……！

no mention of 'lay down'. If 'no choice but to die' feels plain, then I'd add smth like helplessly.

Reply:
Yeah, my intent here was a more idiomatic translation. "Lay down and die" is an EN set phrase, and since he's using 息絶える instead of plain old die I went with something a little fancier.